CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v1.6.2 - 2021-04-26
+
+### Bug fixes
```

Comment:
```suggestion
[full changelog](https://github.com/aiidateam/aiida-core/compare/v1.6.1...v1.6.2) | [GitHub contributors page for this release](https://github.com/aiidateam/aiida-core/graphs/contributors?from=2021-03-31&to=2021-04-26&type=c)

This patch release is intended to include final fixes/features for the `1.x` series, before the new file repository implementation is merged into the `develop` branch (see [#4345](https://github.com/aiidateam/aiida-core/pull/4345)), and work towards finalising a `2.x` release commences.

### Bug fixes
```

Reply:
Done

Reply:
Sorry to meddle, but I think saying these are the final fixes for `1.x` may transmit the wrong idea for some people (as we would still be doing any important bug fixes that came up). Perhaps we could write something more specific of why we released this and less "future oriented", like:

```suggestion
This patch release is mainly intended to include some important fixes for the nested namespaces of output links in workflows.
It also features a fix for the `gotocomputer` CLI command, an improvement in the commands that require the loading of the database, an additional option for the `LsfScheduler`, and some updates of the documentation
See the list below for more information.

These represent the latest work done in the code, being released now before merging in some major changes related to the file repository.

### Bug fixes
```

Reply:
Nope disagree with the suggestion

Reply:
I personally agree with @ramirezfranciscof - the changelog should not contain information on the intended future, but on what has been changed. We can still convey that this is supposed to be the last 1.x release in the [wiki page with the roadmap](https://github.com/aiidateam/aiida-core/wiki/AiiDA-release-roadmap) (which anyway needs to be updated), but I wouldn't write "in stone" in a changelog - e.g. if the work towards 2.0 takes months, there's a large probability that we might have to release 1.6.3 with a few important bug fixes. 

I think the sentence of Francisco is OK, but OK to rephrase it a bit or we might even remove it, I don't think it's critical for the changelog.

Reply:
I still disagree, but it's not a hill worth dying on

Reply:
I also tend to agree with @giovannipizzi and @ramirezfranciscof and propose we remove any explicit mentions of timelines. I am OOO atm but we will update this shortly (also including the other suggestion by @ramirezfranciscof ) and then we can release it.

Reply:
ok thats fine, but then please re-write it in a different way, because most of the added text is superfluous.

Reply:
I meant that I agreed with the sentiment that the timeline/roadmap shouldn't really be in the changelog. I agree that the replacement text is essentially superfluous. I don't really see why we would need any text to justify the patch. It is a patch release. It contains bug fixes. It merits to be released. The list is there to say _what_ changed.